phpunit.xml.dist

```diff
@@ -37,6 +37,9 @@
         <env name="APP_ENV" value="test"/>
         <env name="SHELL_VERBOSITY" value="-1" />
         <env name="TPAY_API_URL" value="http://localhost:4000" />
+        <env name="JWT_SECRET_KEY" value="%kernel.project_dir%/config/jwt/private-test.pem" />
```

Comment:
Since this is a test application, I believe we can use the same JWT keys across all environments.

Reply:
I had some issues to run it locally, if the build will be green I would prefer to leave it for now